<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createCamMatrixDialog</class>
 <widget class="QDialog" name="createCamMatrixDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Camera Matrix</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>371</width>
     <height>371</height>
    </rect>
   </property>
   <property name="title">
    <string>Import photos</string>
   </property>
   <widget class="QLabel" name="camMatPicDirLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>351</width>
      <height>311</height>
     </rect>
    </property>
    <property name="text">
     <string>Please import at least 10 photos shot using the same camera. 
Each photo should contain one and the same chessboard.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>23</y>
      <width>351</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="camMatImportButton">
       <property name="text">
        <string>Import</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>368</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Camera Model</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="camMatModelLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="createCamMatButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>452</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Create!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="camMatCloseButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>460</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
